website/src/components/inputTable/ColumnsTable.jsx

```diff
@@ -18,7 +18,7 @@ const ColumnsTable = (props) => {
                     className="tr"
                 >
                     <td key={props.keyObj.toString([props.fieldKey, index, 'key'])} className="td">
-                        <input
+                        <TextField
```

Comment:
Have we mentioned replacing all `TextField` with `InputString` is it on this pr?

Reply:
it's in the UI revamp pr